src/include/platform/internal/GenericConfigurationManagerImpl.h

```diff
@@ -118,7 +118,9 @@ class GenericConfigurationManagerImpl : public ConfigurationManager
 protected:
 #if CHIP_ENABLE_ROTATING_DEVICE_ID && defined(CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID)
     chip::LifetimePersistedCounter<uint32_t> mLifetimePersistedCounter;
-    uint8_t mRotatingDeviceIdUniqueId[kRotatingDeviceIDUniqueIDLength] = CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID;
+    uint8_t mRotatingDeviceIdUniqueId[CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID_LENGTH] =
+        CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID;
+    size_t mRotatingDeviceIdUniqueIdLength = kRotatingDeviceIDUniqueIDLength;
```

Comment:
Should't that be the actual size of `CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID`?

Reply:
That is what the constant `kRotatingDeviceIDUniqueIDLength` is:
https://github.com/project-chip/connectedhomeip/blob/48860e29fca4bef88c433dfa0436334c7870d112/src/include/platform/ConfigurationManager.h#L80

Reply:
Reopen this if you meant that setting mRotatingDeviceIdUniqueIdLength to CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID_LENGTH is preferable instead of going through kRotatingDeviceIDUniqueIDLength

Reply:
If they're the same thing, this seems fine.  The naming could use some cleanup to make it clear which things are lengths of actual things vs which things are max-lengths, but....